src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -135,7 +135,7 @@ protected function addCookieToResponse($request, $response)
         $response->headers->setCookie(
             new Cookie(
                 'XSRF-TOKEN', $request->session()->token(), time() + 60 * 120,
-                $config['path'], $config['domain'], $config['secure'], false
+                $config['path'], $config['domain'], Arr::get($config, 'secure', false), Arr::get($config, 'http_only', true)
```

Comment:
Wouldn't this introduce a breaking change/behaviour change?

Currently, `http_only` is set to `false`, but with this change (and without updating `laravel/laravel` configuration), `http_only` would default to `true`.